entproto/cmd/protoc-gen-entgrpc/template/method_mutate.tmpl

```diff
@@ -34,9 +34,12 @@
     {{- range .G.FieldMap.Edges }}
         {{- if .EntEdge.Unique }}
             {{- $varName := camel (print $reqVar  "_"  .EntEdge.Name) -}}
-            {{- $id := print $reqVar ".Get" .PbStructField "().Get" .EdgeIDPbStructField "()" -}}
-            {{- template "field_to_ent" dict "Field" . "VarName" $varName "Ident" $id }}
-            m.Set{{ .EntEdge.StructField }}ID({{ $varName }})
+            {{- $id := print $reqVar ".Get" .PbStructField "().Get" .EdgeIDPbStructField "()" }}
```

Comment:
I like to use `printf` in this cases because I think it is more readable. Personal choice.

Reply:
Agree with @masseelch